Sources/Apodini/Semantic Model Builder/REST/RESTInterfaceExporter.swift

```diff
@@ -165,7 +164,16 @@ class RESTInterfaceExporter: InterfaceExporter {
                 return nil
             }
 
-            return try request.content.decode(Type.self, using: JSONDecoder())
+            #warning("""
+                     A Handler could define multiple .content Parameters. In such a case the REST exporter would
+                     need to decode the content via a struct containing those .content parameters as properties.
+                     This is currently unsupported.
+                """)
+            do {
+                return try request.content.decode(Type.self, using: JSONDecoder())
+            } catch {
+                return nil
+            }
```

Comment:
I would suggest to add a `try?` operator here:
```suggestion
        return try? request.content.decode(Type.self, using: JSONDecoder())
```